src/styleguide/_patterns/05-pages/00-homepage.json

```diff
@@ -36,31 +36,21 @@
           },
           {
             "paragraph": "To learn more, check out our <a href='https://medium.com/buildit/'>blog</a>."
+          },
+          {
+            "paragraph": "Here are some of the clients we have worked with:"
```

Comment:
as I was trying to explain previously, this should belong to the following section, as it is now it does not make any sense from a semantic point of view.